frappe/boot.py

```diff
@@ -417,3 +419,34 @@ def get_translatable_doctypes():
 		"Property Setter", {"property": "translate_link_fields", "value": "1"}, pluck="doc_type"
 	)
 	return dts + custom_dts
+
+
+def load_country_doc(bootinfo):
```

Comment:
This is not new addition to boot, it's moved from ERPNext. Doesn't make sense to do this in ERPNext if framework depends so tightly on it.